---
title: "Oxford Nanopore Sequencing Analysis"
excerpt: "Benchmarking multiple ONT basecallers, MSA from multiple basecalled reads to improve mapping accuracy & reduce soft-clipped regions<br/><img src='/images/nanofusion.png' width="70%">"
collection: portfolio
---
##### Budapest University of Technology and Economics (BME) (Study Abroad). Advisor: Péter Sárközy

<center><img src='/images/nanofusion.png' width="75%"></center><br><br>
This project involves benchmarking multiple Oxford Nanopore Technology (ONT) sequence basecallers, and aligning the basecalled reads to reduce soft-clipped regions and improve mapping accuracy. [[Code]](https://github.com/YinuoJin/NanoFusion)<br><br>
Oxford Nanopore Sequencing, a representative third-generation sequencing technology, can produce extremely long sequencing reads, which by its nature could help with the detection of structural variations from the sequence results. However, the decoded sequences from the electrical signals are error-prone, and several methods based on deep learning have been designed to cope with this problem. I benchmarked the performances of multiple recently proposed basecallers, including [Albacore](https://denbi-nanopore-training-course.readthedocs.io/en/stable/basecalling/basecalling.html), [Guppy](https://nanoporetech.com/nanopore-sequencing-data-analysis), [Chiron](https://github.com/haotianteng/Chiron), [Scrappie](https://github.com/nanoporetech/scrappie), and [Flappie](https://github.com/nanoporetech/flappie), with NA12787 chr1. <br><br>
I initially attempted to align the decoded results from different basecallers from the same electrical signals to autocorrect wrongly inferred bases. When two sequences are involved, pairwise semi-global alignment is performed; when three sequences are involved, 3-dimensional sequence alignment is performed; when more than three sequences are involved, I applied [PASTA](https://github.com/smirarab/pasta), a state-of-art multiple sequence alignment (MSA) is applied instead of simple Smith-Waterman like algorithm. When indels appear in the aligning results, "insertions" or "deletions" are determined by the indel ratio of the specific basecaller mapped to the reference sequence. When mismatches appear, the consensus character is determined by either the majority call (if exists) or from the basecalling quality indicated by the PHRED scores of each basecaller at that site. <br><br>

However, the read accuracy obtained by the consensus output didn't consistently outperform the best single basecalling result, due to high overlapping probability distribution of error basecalling sites among all methods (e.g. Homopolymers), and the obvious performance differences between the methods (Flappie consistently performs better than other methods). Nevertheless, there is approximately 5-6% improvement on mapping accuracy due to <u>the trimmed artifact noises</u>. I finally decided to bias the results of Flappie, whereas applying MSA to reduce the artifact noises on two sides of each read.
