<!DOCTYPE HTML>
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

  <title>Yinuo Jin</title>

  <meta name="author" content="Yinuo Jin">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <link rel="stylesheet" type="text/css" href="stylesheet.css">
  <link rel="icon" type="image/png" href="images/seal_icon.png">
</head>

<body>
  <table style="width:100%;max-width:900px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
    <tr style="padding:0px">
      <td style="padding:0px">
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr style="padding:0px">
            <td style="padding:2.5%;width:63%;vertical-align:middle">
              <p style="text-align:center">
                <name>Yinuo Jin</name>
              </p>
              <p>I'm a fourth-year undergraduate student at Columbia University majoring in Computer Science.
              </p>
              <p>
              I'm very fortunate to be advised by <a href="https://www.engineering.columbia.edu/faculty/itsik-peer">Prof. Itsik Pe'er</a> and <a href="https://www.bme.columbia.edu/faculty/elham-azizi">Prof. Elham Azizi</a> and have been working on multiple projects in their labs. Before transferring to Columbia, I studied Physics at Colgate University. In Spring 2019, I attended a semester-long CS study abroad program at <a href="https://www.ait-budapest.com/syllabuses/computational-biology-and-medicine">AIT-Budapest</a> with focus on Compbio, during which I was advised by Prof. Péter Sárközy at <a href="http://bioinformatics.mit.bme.hu/">Computational Biomedicine Workgroup</a>, Budapest University of Technology and Economics (BME).
              </p>
              <p style="text-align:center">
                <a href="yj2589@columbia.edu">Email</a> &nbsp/&nbsp
                <a href="data/Yinuo_Jin_CV.pdf">CV</a> &nbsp/&nbsp
                <a href="https://github.com/YinuoJin/">Github</a>
              </p>
            </td>
            <td style="padding:2.5%;width:40%;max-width:40%">
              <a href="images/profile.png"><img style="width:100%;max-width:100%" alt="profile photo" src="images/profile.png" class="hoverZoomLink"></a>
            </td>
          </tr>
        </tbody></table>

        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
            <tr>
            <td style="padding:20px;width:100%;vertical-align:middle">
              <heading>Research & Projects</heading>
              <p>
                My research interests lie in Computational Biology, particularly in applying machine learning methods on high-throughput sequencing data to understand the underlying biological questions.
              </p>
            </td>
          </tr>
        </tbody></table>

        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/picasso.png" width="320" height="196">
            </td>
            <td width="75%" valign="middle">
                <papertitle>Phylogenetic Inference from Copy Number Variations (CNVs) in scRNA-seq data</papertitle>
              <br>
              <p>We propose PICASSO (<b>P</b>hylogenetic <b>I</b>nference from <b>C</b>opy Number <b>A</b>lterations in <b>S</b>crna<b>S</b>eq <b>O</b>bservations), a novel iterative approach to infer subclonal structure from tumor scRNA-seq data based on CNV predictions.
                 Besides the model itself, I designed a fully automated simulation pipeline facilitated by <a href="https://github.com/Oshlack/splatter">Splatter</a>, generating large-scale, synthetic observation matrices from the real scRNA-seq data.
              </p>
            </td>
          </tr>

          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/mrna.png" width="320" height="240">
            </td>
            <td width="75%" valign="middle">
                  <papertitle>Transfer learning framework for cell segmentation with incorporation of geometric features</papertitle>
                <br>
                <strong>Yinuo Jin*</strong>, Alexandre Toberoff*, and Elham Azizi<br>NeurIPS 2020, Workshop on Learning Meaningful Representations of Life (LMRL)
                <br>
                <a href="https://drive.google.com/file/d/1XZwqcILo9p7AQ-dK_0w0SWzdB_KwU2pg/preview">Paper</a> / <a href="data/LMRL_poster.pdf">Poster</a> / <a href="https://github.com/YinuoJin/cell-segmentation">Code</a>
              <br>
              <p>We propose a cell segmentation framework incorporating shape-aware features in a deep learning model, with multi-level watershed and morphological post-processing steps. Our results show that incorporation of geometric features improves generalizability to segmenting cells in in situ tissue images, using solely in vitro images as training data.
              </p>
            </td>
          </tr>

          <tr>
            <td style="padding:20px;width:25%;vertical-align:middle">
              <img src="images/asr.png" width="320" height="190">
            </td>
            <td width="75%" valign="middle">
                <papertitle>Extending a Feature-Rich Python Library
for Ancestral Sequence Reconstruction</papertitle>
              <br>
              <a href="data/asr_treetime.pdf">Report</a> / <a href="https://github.com/YinuoJin/treetime">Code</a>
              <p>We implemented multiple recently proposed algorithms for Ancestral Sequence Reconstruction (ASR), extending features to <a href="https://github.com/neherlab/treetime">TreeTime</a>, a phylogenetic analysis toolkit. <br><br>
                The original vanilla ASR implementation in TreeTime assumed fixed substitution rate and independent evolution process at different sites along any given sequence during the reconstruction. We implemented Among-site Rate Variations (ASRV) & site-specific constraints based on the inferred solvent accessibility or secondary structure for protein sequences. The modifid models outperform the vanilla ASR, for either the actual accuracy or the log-likelihood scores.
              </p>
            </td>
          </tr>

        </tbody></table>
      </td>
      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        <tr>
          <td style="padding:0px">
            <p style="text-align:center">
              Source code from <a href="https://jonbarron.info/">Jon Barron's personal website</a>
            </p>
          </td>
        </tr>
      </tbody></table>
    </tr>
  </table>
</body>

</html>
